resources/wif/README.md

```diff
@@ -0,0 +1,57 @@
+# How WIF templates work - high level
+
+When a customer creates an OSD-GCP cluster using Workload Identity Federation,
+a wif_config object is made. The content of that wif_config object lists the
+service accounts and permissions needed for the cluster's deployment and
+operation. This criteria is defined here, and then passed to OCM where it hosts
+an API endpoint for clients to query. The API provided by Clusters Service
+watches for changes in [this](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/ocm/shared-resources/common.yml)
+file. When this file changes a new [ConfigMap](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/services/ocm/gcp-wif-template.configmap.yaml)
+is templated which Clusters Service pods (OCM) will read.
+
+## How to update WIF templates
+
+When Service Accounts, Roles and/or Permissions need to be updated, execute the following. 
```

Comment:
Can permissions be updated? AFAIU these permissions are consumed by the ocm-cli and changing them would also require customers to re-create the roles. I think for STS we don't update the permissions during a y stream.

Reply:
That's the only time that STS permissions get updated on AWS clusters; as part of a y-stream upgrade.

Reply:
Might have expressed it wrong but that's what I meant. The references piece of text sounds like upgrading within a y stream is a possibility.

Reply:
I reworded the sentence slightly. Once I've updated the GAP analysis document for WIF, i will also link that here.